app/uploaders/logo_svg_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class LogoSvgUploader < BaseUploader
+  # This Uploader is being used by a Data Update Script
+  # lib/data_update_scripts/20211208074428_migrate_logo_svg_data.rb
+  # to convert logo_svg contents to a png file.
+
+  STORE_DIRECTORY = "uploads/logos/".freeze
+  EXTENSION_ALLOWLIST = %w[svg png].freeze
+  IMAGE_TYPE_ALLOWLIST = %i[svg png].freeze
+  CONTENT_TYPE_ALLOWLIST = %w[image/svg+xml image/png].freeze
+
+  process convert: "png"
```

Comment:
We set a process callback, which will call that method any time a file is uploaded. Thereafter the MiniMagick convert method which changes the image encoding format to the given format.